### PR TITLE
Add Choose a license docs from NLeSC guide

### LINKED ISSUE #868: Choosing software license and examples
The choosing of a software license is now a link to https://choosealicense.com/. 
On the NLeSC guide there is a [document]( https://guide.esciencecenter.nl/intellectual_property_and_licensing/software_licenses.html) which explains the different types of licenses similar to https://choosealicense.com/ . We also have some [examples](https://guide.esciencecenter.nl/intellectual_property_and_licensing/examples_libraries.html) and [here](https://guide.esciencecenter.nl/intellectual_property_and_licensing/more_examples.html), how to combine software with different license types.

### Summary

We would like to add a document to the [Open source software chapter](https://the-turing-way.netlify.com/open_research/02/opensourcesoftware.html) to help you choose a software license and add a document which shows some of the difficulties choosing the right license when combining software.
The new document would replace the existing choosealicense link. The choosealicense link will be included in the new document. 

### What needs to be done?

- [x] Add [choose a license doc](https://guide.esciencecenter.nl/intellectual_property_and_licensing/software_licenses.html) from NLeSC guide to Turing Way
- [x] Add example of how licenses interact: [for libraries](https://guide.esciencecenter.nl/intellectual_property_and_licensing/examples_libraries.html) and [more examples](https://guide.esciencecenter.nl/intellectual_property_and_licensing/more_examples.html) to Turing Way
* [x] Add choose https://choosealicense.com/ to new document
* [x] Replace mentions of https://choosealicense.com/ by new document
* [x] Remove NLeSC specific way of working, by proofreading it as a non-NLeSC employee
* [x] Link to library home pages
* ~~[x] Update references.bib with things from new document~~
### Who can help?

* *Lorem ipsum dolor sit amet, consectetur adipiscing.*
* *Lorem ipsum dolor sit amet, consectetur adipiscing.*

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by sverhoeven**
### Summary

Fixes #868

### List of changes proposed in this PR (pull-request)

* *Lorem ipsum dolor sit amet, consectetur adipiscing.*
* *Lorem ipsum dolor sit amet, consectetur adipiscing.*

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: 
    * @sverhoeven, @c-martinez and @lourensveen (original author of chapter in NLeSC guide)

### CODE REVIEW

**Comment by malvikasharan** [comment]
@sverhoeven  Is it work in progress [WIP] or ready for review.

**Comment by sverhoeven** [comment]
Still a work in progress

**Comment by LourensVeen** [comment]
Maybe there should be a bit of a discussion on whether replacing the link to ChooseALicense with this giant pile of text I wrote is actually an improvement. Reading back what I wrote years ago, I think it's actually a good explanation of what happens when you combine software from different places under different licenses, and I'm not aware of any other place where it's explained like this (including case law, a judge may disagree with this explanation actually, but that goes for any explanation currently). Still, I spent quite a lot of time on this, and it's the only point of view I've found that matches the comments and advice I've found from experts, e.g. FSF lawyers. So I think it's a potentially useful contribution to the internet.

However, most users of both the NLeSC Guide and The Turing Way will be scientists who don't care about licenses at all, but will be willing to spend five minutes putting a license on their code because someone told them they should. They're unlikely to read this whole explanation. Of course, that may mean that they'll miss a detail and do something that legal-technically doesn't work, but there's probably a lot of open source code out there about which that can be said.

Putting it in or leaving it out is both fine with me, but I think this should be discussed.

**Comment by LourensVeen** [comment]
The diagrams are eScience cyan. I'm not sure if the original SVGs are in the Guide repo, but I should have them still, so just let me know if you want them in a different colour.

File: book/content/open_research/02/softwarelicenses.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Software licenses
 
+*Disclaimer: Good legal advice is timely, specific, and given by an expert; this chapter is none of these. It was written by an engineer, not by a lawyer, and it's a heavily simplified overview of a very complex field. The intent is to give you an overview of the basics, so that you will know when to check whether something you want to do has potential legal ramifications. Don't make any important decisions based solely on the contents of this chapter.*
```
Comment (malvikasharan) [inline book/content/open_research/02/softwarelicenses.md]: Please move disclaimers to the end of the chapter or rephrase it to be added right after the next two statements defining the license.

File: book/content/open_research/02/softwarelicenses.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
 There are many software licenses in existence. Many of those allow the licensee to do very little, but some give you more freedom to use and re-use the licensed software.
```
Comment (malvikasharan) [inline book/content/open_research/02/softwarelicenses.md]: Can we add a sentence on why do we need licenses as a motivation for this chapter? Or have written that somewhere already?

**Comment by malvikasharan** [comment]
@c-martinez and @sverhoeven Do you think that your license chapter can be moved to its own chapter? I think we need to talk about license in general (for e.g. in context of reproducible research) and not only in open science. Though of course, we need to add a section on how to choose a license when the project can not be opened due to sensitivity or data privacy reasons. What do you think?

**Comment by LourensVeen** [comment]
In Europe at least, if you want to share privacy-sensitive data with a processor, then you need to require the processor to do various things through a contract. That's based on privacy law (the GDPR) and contract law however, and therefore independent legally of copyright licensing. Also, data sets may or may not be copyrightable, and may additionally fall under database rights law, depending on jurisdiction. So I would recommend separate sections on licensing software (which would be the present part) and on distributing data (which would talk about Creative Commons and the Open Database Licenses, and include something about privacy as well).

**Comment by c-martinez** [comment]
Hi @malvikasharan -- sure, if you think this could be a chapter on its own, we are happy to move it. Following @LourensVeen 's comment, perhaps we should make a chapter called "Licensing" with two main sections: "Software" for the content in this PR, and "Data" with content to be filled at a later stage.

Does that sound like a good plan?

**Comment by malvikasharan** [comment]
@c-martinez and @sverhoeven I have sent a PR to your branch. 

Can we add software documentation license in the software license sub-chapter or have that as an extra chapter?

**Comment by mkuzak** [comment]
There was conflict in the toc.yml file. @c-martinez can you merge the PR from @malvikasharan ? that should resolve it :)

**Comment by c-martinez** [comment]
Hey @mkuzak -- I merged @malvikasharan 's PR . I still had to fix another conflict, but now it is ready (and merged!) Thanks @malvikasharan !
